### DISCUSSION TITLE
Unable to create custom card Nova 3

### DISCUSSION THREAD

**Discussion by Cathari** [laravel/nova-issues#4526] [Q&A]
- Laravel Version: 9.19.0
- Nova Version: 3.32.0
- PHP Version: 8.1.5
- Vue Version: 3*

### Description:
How can I create a custom card on Nova 3?
After creating, compiling and adding custom card to the resource...
![изображение](https://user-images.githubusercontent.com/54035329/177037501-a9a156dc-4401-4d90-88ff-3943dc6423b3.png)

### Detailed steps to reproduce the issue on a fresh Nova installation:
[https://flywheel.co/articles/how-to-create-custom-laravel-nova-cards](url)

**Comment by crynobone** [comment]
Nova 3 uses Vue 2 and not Vue 3
